woocommerce/payment-gateway/assets/js/frontend/sv-wc-payment-gateway-payment-form.coffee

```diff
@@ -64,6 +64,12 @@ jQuery ( $ ) ->
 		# Returns nothing.
 		handle_checkout_page: ->
 
+			# make sure payment fields initialized upon method selection
+			$( document.body ).on( 'payment_method_selected', =>
```

Comment:
what prompted this change? The form JS should have already been loaded, via the `updated_checkout` even that calls `this.set_payment_fields` below.

If I revert this change (and fix the issues from my other comments) then the form seems to load okay. Was there a case where you saw it not working?

Reply:
Without this change, the form loads but it doesn't process fields upon submit ... it doesn't load payment fields values, ex => https://cloud.skyver.ge/nOulyjbo

So I thought what other checkout event I can listen to to make sure fields get initialized correctly, so I thought using the `payment_method_selected` is a good idea, so with that change the fields get processed normally.

That said, now when I am testing it, I got a different behavior than yesterday! which I swear it was working!